Subject: [PATCH] winpr-makecert: Build with install RPATH

As we are installing "winpr-makecert" manually, we do want
to refer to the library in DESTDIR, not the one resting in
the build tree.

Update it for 2.0.0.

Signed-off-by: Manuel Bachmann <manuel.bachmann@iot.bzh>
Signed-off-by: Zheng Ruoqin <zhengrq.fnst@cn.fujitsu.com>
---
Upstream-Status: Pending

 winpr/tools/makecert-cli/CMakeLists.txt | 3 +++
 1 file changed, 3 insertions(+)

Index: git/winpr/tools/makecert-cli/CMakeLists.txt
===================================================================
--- git.orig/winpr/tools/makecert-cli/CMakeLists.txt
+++ git/winpr/tools/makecert-cli/CMakeLists.txt
@@ -44,6 +44,9 @@ set(${MODULE_PREFIX}_LIBS winpr-tools)
 
 target_link_libraries(${MODULE_NAME} ${${MODULE_PREFIX}_LIBS} winpr)
 
+set_target_properties(${MODULE_NAME} PROPERTIES
+       BUILD_WITH_INSTALL_RPATH 1)
+
 set_property(TARGET ${MODULE_NAME} PROPERTY FOLDER "WinPR/Tools")
 
 install(TARGETS ${MODULE_NAME} DESTINATION ${CMAKE_INSTALL_BINDIR} COMPONENT tools EXPORT WinPRTargets)
